pkg/sampling/strategystore/constants.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
yes, the `pkg` is rather suspicious for this. It is meant for things not directly related to Jaeger. The simplest approach would be to define the storage interface in `cmd/collector/sampling/strategystore/interface.go`

Reply:
done, should I have the static/adaptive implementations in plugin or should I move the whole thing over to cmd/collector/app?